mscore/musescore.cpp

```diff
@@ -6880,6 +6880,9 @@ int main(int argc, char* av[])
 #endif
 
       QApplication::setDesktopSettingsAware(true);
+#if QT_VERSION >= 0x050700 && defined(Q_OS_LINUX)
```

Comment:
Sure this needs Qt 5.7 (Edit: I see, yet it is)? Anyway, this is the minimum required for MuseScore 3,x anyhow IIRC, so no need to check for.

Reply:
Sure this needs Qt 5.7 (Edit: I see, yes, it does)? Anyway, the minimum required for MuseScore 3,x is Qt 5.8 anyhow, so no need to check for this (and even another PR from @mirabilos, #4565, only goes back to Qt 5.7)